infra/bootstrap.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash -e
+
+#
+# Do not prompt for answers
+export DEBIAN_FRONTEND=noninteractive
+export LC_ALL="en_US.UTF-8"
+export LC_CTYPE="en_US.UTF-8"
+export LC_ALL=C
+sudo dpkg-reconfigure --frontend=noninteractive locales
+
+
+# Add Third-party repository for Python 3.6 distribution 
+# Ubuntu 16.04 does not provide via repos.
+sudo add-apt-repository ppa:jonathonf/python-3.6 -y
+sudo apt-get -y update
+
+sudo DEBIAN_FRONTEND=noninteractive apt-get -y -o Dpkg::Options::=\"--force-confdef\" -o Dpkg::Options::=\"--force-confold\" dist-upgrade
+
+sudo apt-get -y install \   # Install basic utilities
+    awscli \                # AWS CLI
+    curl \                  # Downloading stuff
+    git \                   # Pull code for deployment
+    ruby \                  # Dependency for AWS tools
+    unzip \                 # AWS tools are usually zip'd
+    wget \                  # Downloading stuff
+    grub kpartx             # Dependencies for ec2-ami-tools
+
+sudo apt-get -y install \      # Install all things Python
+    python3.6 python3.6-dev \  # Python 3.6 from the third-party repo
+    python3.6-venv \           # Python virtualenv
+    python-pip                 # Default python pip (why?)
+
+sudo apt-get -y install \        # Install Mapping related stuff
+    libgdal1-dev \               # Geospatial Data Abstraction Library (GDAL)
+    libgeos-3.5.0 libgeos-dev \  # Geometry engine
+    libproj9 libproj-dev \       # Cartographic projects & translations between CRS
+    libxml2 libxml2-dev \        # XML parsing and such
+    libjson-c-dev                # Construct JSON objects in C
+
+sudo apt-get -y install \
+    postgresql-9.5 \             # PostgreSQL server
```

Comment:
I think we are already running on postgresql 11 (also on live!?).

Reply:
Then we can change this. But Ubuntu 16.04 does not ship PostgreSQL 11. Ubuntu 18.04 ships PostgreSQL 10. Debian Buster will have PostgreSQL 11. Either way, not sure why we need the server on the instance if we use RDS?

Reply:
Right. If this is not necessary. Then we should probably omit.

Reply:
I'll leave it in for now until I figure out why it was put in in the first place.